app/helpers/sessions_helper.rb

```diff
@@ -9,6 +9,10 @@ def remember user
     cookies.permanent[:remember_token] = user.remember_token
   end
 
+  def current_user?(user)
+    user &. == current_user
```

Comment:
viết ntn có ý nghĩa gì đấy ?

Reply:
" &. "  : là toán tử điều hướng an toàn , làm cho đoạn code nhỏ gọn hơn,cú pháp chỉ bỏ qua nilnhưng nhận ra false!